src/pubtools/_impl/tracing.py

```diff
@@ -33,11 +41,18 @@ def get_trace_wrapper():
     """return a global trace wrapper instance"""
     global TRACE_WRAPPER
     if TRACE_WRAPPER is None:
-        TRACE_WRAPPER = TracingWrapper()
+        if OPENTELEMETRY_AVAILABLE:
+            TRACE_WRAPPER = ServerTracingWrapper()
+        else:
+            log.warning(
```

Comment:
I would prefer not to WARN here, the problem is that we don't really have control over where get_trace_wrapper is called, and how many times.

This could easily result in several duplicate warnings produced every time certain tools are run, confusing the users. Note, if a tool fails for *any reason*, and it output some warnings prior to the failure, there's usually going to be some users who assume the warned-about problem caused the failure.

Can we perhaps do one of these?

- make it DEBUG instead of WARN - that means it's still possible to tell whether this code path is taken by increasing the log level.
- or keep it as WARN but only if OTEL_TRACING environment variable has been set to true (the same condition used to initialize `_enabled_trace`).  Because this would indicate that someone is explicitly asking for OTEL tracing in an environment where it's not available.

Reply:
I've reworked this change to just disable the tracing if OTEL isn't available, this is much more straight forward. I've changed the log to debug. I think it's important to have some logging here, as we're silencing an exception.